src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleMethodSymbol.vb

```diff
@@ -0,0 +1,144 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports System.Globalization
+Imports System.Threading
+Imports Microsoft.CodeAnalysis
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
+    ''' <summary>
+    ''' Represents a method of a tuple type (such as (int, byte).ToString())
+    ''' that is backed by a method within the tuple underlying type.
+    ''' </summary>
+    Friend Class TupleMethodSymbol
+        Inherits WrappedMethodSymbol
+
+        Private _containingType As TupleTypeSymbol
+
+        Private _underlyingMethod As MethodSymbol
+
+        Private _typeParameters As ImmutableArray(Of TypeParameterSymbol)
+
+        Private _lazyParameters As ImmutableArray(Of ParameterSymbol)
+
+        Public Overrides ReadOnly Property IsTupleMethod() As Boolean
+            Get
+                Return True
+            End Get
+        End Property
+
+        Public Overrides ReadOnly Property TupleUnderlyingMethod() As MethodSymbol
+            Get
+                Return Me._underlyingMethod.ConstructedFrom
+            End Get
+        End Property
+
+        Public Overrides ReadOnly Property UnderlyingMethod() As MethodSymbol
+            Get
+                Return Me._underlyingMethod
+            End Get
+        End Property
+
+        Public Overrides ReadOnly Property AssociatedSymbol() As Symbol
+            Get
+                Return Me._containingType.GetTupleMemberSymbolForUnderlyingMember(Of Symbol)(Me._underlyingMethod.ConstructedFrom.AssociatedSymbol)
+            End Get
+        End Property
+
+        Public Overrides ReadOnly Property ContainingSymbol() As Symbol
+            Get
+                Return Me._containingType
+            End Get
+        End Property
+
+        Public Overrides ReadOnly Property ExplicitInterfaceImplementations() As ImmutableArray(Of MethodSymbol)
+            Get
+                Return Me._underlyingMethod.ConstructedFrom.ExplicitInterfaceImplementations
+            End Get
+        End Property
+
+        Public Overrides ReadOnly Property Parameters() As ImmutableArray(Of ParameterSymbol)
+            Get
+                Dim isDefault As Boolean = Me._lazyParameters.IsDefault
```

Comment:
fixed

---
In reply to: [71548657](https://github.com/dotnet/roslyn/pull/12596#discussion_r71548657) [](ancestors = 71548657)